CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 1.0.0-rc.0 / 2016-07-14
+
+* [CHANGE] Remove deprecated query language keywords
+* [CHANGE] Change Kubernetes SD to require specifying Kubernetes role
+* [CHANGE] Use service address in Consul SD if available
+* [CHANGE] Standardize all Prometheus internal metrics to second units
+* [CHANGE] Remove unversioned legacy HTTP API
+* [CHANGE] Make PromQL vector matching to be ignoring by default
```

Comment:
This is an internal noop change, no need to mention it in the changelog. The `on()` is the feature it enables.